### DISCUSSION TITLE
Workspaces and virtual projects

### DISCUSSION THREAD

**Discussion by mjclarke94** [astral-sh/rye#685] [General]
The ideas behind both of these seem really interesting, but it would be really useful to see examples of these in the wild! Can anyone point me to:

a) An example of these being used for a monorepo type setup
b) Any examples of how to use these for local development with git repos for each individual project

Thanks in advance!

**Comment by jamesbraza** [comment]
+1 to this. I am looking for an example of a repo using `rye` with two Python packages:

- Package `a`
- Package `b`, which depends on package `a`

**Comment by HichemAK** [comment]
+1
